scripts/kwargs-to-map/test/kwargs_to_map/core_test.clj

```diff
@@ -0,0 +1,292 @@
+[line
+ :size  \"3px\"
+ :color \"red\"]"))
+
+(def label-example (z/of-string "
+[label
+ :label  \"This is the label\"
+ :style  {:color \"red\"}
+ :class \"class-name\"]"))
+
+(def title-example (z/of-string "
+[title
+ :label  \"This is the label\"
+ :level  :level1
+ :style  {:color \"red\"}
+ :class \"class-name\"]"))
+
+(def basic-button-example (z/of-string "
+[button
+ :label    \"This is the label\"
+ :style    {:color \"red\"}
+ :on-click nil
+ :class    \"class-name\"]"))
+
+(def row-button-example (z/of-string "
+[row-button
+ :md-icon-name  \"zmdi-plus\"
+ :style         {:color \"red\"}
+ :on-click      nil
+ :class         \"class-name\"]"))
+
+(def circle-icon-example (z/of-string "
+[md-circle-icon-button
+ :md-icon-name  \"zmdi-plus\"
+ :style         {:color \"red\"}
+ :on-click      nil
+ :class         \"class-name\"]"))
+
+(def md-icon-button-example (z/of-string "
+[md-icon-button
+ :md-icon-name  \"zmdi-plus\"
+ :size          \"1\"
+ :style         {:color \"red\"}
+ :on-click      nil
+ :class         \"class-name\"]"))
+
+(def info-button-example (z/of-string "
+[info-button
+ :info  \"Some information to show\"
+ :width \"200px\"
+ :style {:color \"red\"}
+ :class \"class-name\"]"))
+
+(def hyperlink-example (z/of-string "
+[hyperlink
+ :label    \"Hyperlink text\"
+ :on-click nil
+ :style    {:color \"red\"}
+ :class    \"class-name\"]"))
+
+(def hyperlink-href-example (z/of-string "
+[hyperlink-href
+ :label    \"Hyperlink text\"
+ :href     \"link.com\"
+ :style    {:color \"red\"}
+ :class    \"class-name\"]"))
+
+(def alert-box-example (z/of-string "
+[alert-box
+  :src        (at)
+  :alert-type :none
+  :style      {:color             \"#222\"
+               :background-color  \"rgba(255, 165, 0, 0.1)\"
+               :border-top        \"none\"
+               :border-right      \"none\"
+               :border-bottom     \"none\"
+               :border-left       \"4px solid rgba(255, 165, 0, 0.8)\"
+               :border-radius     \"0px\"}
+  :body       \"Alert with :body but no :heading (:padding set to 6px).\"
+  :padding    \"6px\"
+  :closeable? true
+  :on-close   #(reset! show-alert6 false)]"))
+
+(def popover-anchor-wrapper-example (z/of-string "
+[popover-anchor-wrapper
+  :showing? showing?
+  :position :right-below
+  :anchor   [button
+             :label    \"Anchor\"
+             :on-click #(swap! showing? not)]
+  :popover  [popover-content-wrapper
+             :title    \"Title\"
+             :body     \"Popover body text\"]]"))
+
+(def h-split-example (z/of-string "
+[rc/h-split :src (at)
+ :panel-1 [left-panel]
+ :panel-2 [right-panel]
+ :size    \"300px\"]"))
+
+(def v-split-example (z/of-string "
+[rc/v-split :src (at)
+ :panel-1 [top-panel]
+ :panel-2 [bottom-panel]
+ :size    \"300px\"
+ :initial-split \"25%\"]"))
+
+(def p-example (z/of-string "
+[p \"This will be ignored\"]"))
+
+(def p-span-example (z/of-string "
+[p-span \"This will be ignored\"]"))
+
```

Comment:
this file actually need assertions ala the add at macro scirpt, not just printing the results to console